packages/guide-cue/src/types.ts

```diff
@@ -37,6 +37,7 @@ type ModifiedTooltipProps = Omit<
   | 'open'
   | 'refEl'
   | 'darkMode'
+  | 'initialOpen'
```

Comment:
It'll get auto-bumped bc of the Tooltip change but not sure what I'd include in a changeset since there's no functional change. We're just ensuring that a change to Tooltip doesn't propagate up.

Reply:
if we don't publish this then `initialOpen` will not be omitted when tooltip is bumped

Reply:
I believe Changesets will auto patch this bc a dependency changed

Reply:
oh that makes sense

Reply:
nvm then